functions/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
     "compilerOptions": {
         "target": "esnext",
-        "module": "esnext",
-        "moduleResolution": "nodenext",
+        "module": "NodeNext",
```

Comment:
It fixes this TS error:

```
Option 'module' must be set to 'NodeNext' when option 'moduleResolution' is set to 'NodeNext'.
```

More importantly, I did it to fix TS errors I had with all imports from `itty-router`, but I can no longer replicate that on master.

It changes the output of TS compiler, but it shouldn't matter since we use TS only for type checking.